staging/src/k8s.io/client-go/examples/create-update-delete-deployment/main.go

```diff
@@ -100,32 +107,63 @@ func main() {
 	//    2. Modify the "result" returned by Create()/Get() and retry Update(result) until
 	//       you no longer get a conflict error. This way, you can preserve changes made
 	//       by other clients between Create() and Update(). This is implemented below:
+	//
+	// See the API Conventions:
+	// https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#concurrency-control-and-consistency
 
 	for {
+		// Retrieve latest version of Deployment before modifying and updating
+		result, err = deploymentsClient.Get("demo-deployment", metav1.GetOptions{})
```

Comment:
Consider using the retry utility function: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/util/retry/util.go#L61

Reply:
Yup it was just brought to my attention that client-go has this utility function, definitely a better way to handle this.. will update with that change

Reply:
I wasn't aware of RetryOnConflict while developing this sample. It looks dope. 👌